src/services/earn/types.ts

```diff
@@ -1,21 +1,21 @@
 import { PermitData, SinglePermitParams } from '@services/permit2';
-import { Address, AmountsOfToken, BigIntish, BuiltTransaction, ChainId, Timestamp, TimeString, TokenAddress } from '@types';
+import { AmountsOfToken, BigIntish, BuiltTransaction, ChainId, Timestamp, TimeString, TokenAddress } from '@types';
 import { ArrayOneOrMore } from '@utility-types';
-import { Hex } from 'viem';
+import { Hex, Address as ViemAddress } from 'viem';
 
 export type IEarnService = {
   getAllowanceTarget(_: {
     chainId: ChainId;
     strategyId: StrategyId;
     depositWith: TokenAddress;
     usePermit2?: boolean;
-  }): Promise<Address | undefined>;
+  }): Promise<ViemAddress | undefined>;
   preparePermitData(_: SinglePermitParams): Promise<PermitData>;
   preparePermissionData(_: {
     chainId: ChainId;
     positionId: PositionId;
     permissions: EarnPermissionSet[];
-    signerAddress: Address;
+    signerAddress: ViemAddress;
```

Comment:
I'm ok with using `ViemAddress` when we return values, but let's not force clients to cast them when sending input params